email-templates/daily/html.ejs

```diff
@@ -0,0 +1,271 @@
+<%
+/**
+ * Copyright ©2015. The Regents of the University of California (Regents). All Rights Reserved.
+ *
+ * Permission to use, copy, modify, and distribute this software and its documentation
+ * for educational, research, and not-for-profit purposes, without fee and without a
+ * signed licensing agreement, is hereby granted, provided that the above copyright
+ * notice, this paragraph and the following two paragraphs appear in all copies,
+ * modifications, and distributions.
+ *
+ * Contact The Office of Technology Licensing, UC Berkeley, 2150 Shattuck Avenue,
+ * Suite 510, Berkeley, CA 94720-1620, (510) 643-7201, otl@berkeley.edu,
+ * http://ipira.berkeley.edu/industry-info for commercial licensing opportunities.
+ *
+ * IN NO EVENT SHALL REGENTS BE LIABLE TO ANY PARTY FOR DIRECT, INDIRECT, SPECIAL,
+ * INCIDENTAL, OR CONSEQUENTIAL DAMAGES, INCLUDING LOST PROFITS, ARISING OUT OF
+ * THE USE OF THIS SOFTWARE AND ITS DOCUMENTATION, EVEN IF REGENTS HAS BEEN ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * REGENTS SPECIFICALLY DISCLAIMS ANY WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE. THE
+ * SOFTWARE AND ACCOMPANYING DOCUMENTATION, IF ANY, PROVIDED HEREUNDER IS PROVIDED
+ * "AS IS". REGENTS HAS NO OBLIGATION TO PROVIDE MAINTENANCE, SUPPORT, UPDATES,
+ * ENHANCEMENTS, OR MODIFICATIONS.
+ */
+%>
+
+<%- include('../_shared/header') %>
+
+<%
+var greeting = 'You have ';
+var parts = [];
+var activityTypeCounts = _.countBy(activities, 'type');
+if (activityTypeCounts.asset_comment) {
+  if (activityTypeCounts.asset_comment === 1) {
+    parts.push('new comments on 1 asset')
+  } else {
+    parts.push('new comments on ' + activityTypeCounts.asset_comment + ' assets')
+  }
+}
+if (activityTypeCounts.asset_comment_reply) {
+  if (activityTypeCounts.asset_comment_reply === 1) {
+    parts.push('1 reply on a comment')
+  } else {
+    parts.push(activityTypeCounts.asset_comment_reply + ' replies on your comments')
+  }
+}
+if (activityTypeCounts.whiteboard_chat) {
+  if (activityTypeCounts.whiteboard_chat === 1) {
```

Comment:
This should be:

- 1 chat message on 1 whiteboard: `a new chat message on a whiteboard`
- multiple chat messages on 1 whiteboard: `x chat messages on 1 whiteboard`
- multiple chat messages on multiple whiteboards: `x chat messages on y whiteboards`